scripts/update-unicode-properties.ts

```diff
@@ -32,6 +32,20 @@ const DataSources = [
         gcValues: "#table-unicode-general-category-values",
         scValues: "#table-unicode-script-values",
     },
+    {
+        url: "https://tc39.es/ecma262/2022/multipage/text-processing.html",
```

Comment:
I think it would be better to keep these all uniform, no?

```suggestion
        url: "https://262.ecma-international.org/13.0",
```

Reply:
Referencing multipage costs less to finishes faster. So I want to refer to multipage.
I would like to unify others, but there is no multipage before ES2021 😢

Reply:
OK let's go with `multipage` as of ES2021 then! 👍